Results biorxiv pmc (#8)
greenelab/annorxiver_manuscript: content/04.results.md

## Results

### Comparing bioRxiv to PubMed Central

#### bioRxiv Repository

![

> BioRxiv has an influx of neuroscience preprints followed by bioinformatics and then microbiology.

Reviewer: ```suggestion
Neuroscience, bioinformatics and microbiology are the most common fields for preprints on bioRxiv.
```

This bar chart depicts the number of preprints that fall into each category of bioRxiv.
](https://raw.githubusercontent.com/greenelab/annorxiver/35d3ea0de3c9c78e3c524736bbaada00928c88fb/biorxiv/exploratory_data_analysis/output/figures/preprint_category.png){#fig:biorxiv_categories}

![

> Majority of bioRxiv's articles report new research findingds.

Reviewer: ```suggestion
Most bioRxiv's preprints report new research findings.
```

This bar char depicts the number of articles categorized based on report findings.
](https://raw.githubusercontent.com/greenelab/annorxiver/94874e0f1e35bd667bf3b7c3dc6416068779444f/biorxiv/exploratory_data_analysis/output/figures/preprint_headings.png){#fig:biorxiv_headings}

> BioRxiv allows users to specify the most appropiate topic for their preprint submission along with the type of finding being reported.

Reviewer: ```suggestion
Users who submit preprints select topic areas and specify the type of finding.
```

> Majority of research findings report new results (Figure {@fig:biorxiv_headings}).

Reviewer: This is out of order with your figures.

> Regarding topic categories, majority of preprints fall within the neuroscience field (Figure {@fig:biorxiv_categories}).

Reviewer: How does this compare with previous descriptions of the composition of bioRxiv, potentially done at earlier times?

### Comparing bioRxiv to PubMed Central

#### bioRxiv Repository

![
Neuroscience and bioinformatics are the two most common topics for preprints on bioRxiv.

> This bar chart depicts the number of preprints that fall into each category of bioRxiv.

Reviewer: ```suggestion
This bar chart depicts the number of preprints that fall into each author-selected topic area.
```

](https://raw.githubusercontent.com/greenelab/annorxiver/35d3ea0de3c9c78e3c524736bbaada00928c88fb/biorxiv/exploratory_data_analysis/output/figures/preprint_category.png){#fig:biorxiv_categories}

![

> Majority of bioRxiv's preprints report new research findings.

Reviewer: ```suggestion
Most of bioRxiv's preprints report new research findings.

```

> This bar char depicts the number of articles categorized based on report findings.

Reviewer: ```suggestion
This bar chart depicts the number of articles categorized based on author-selected article types.
```

](https://raw.githubusercontent.com/greenelab/annorxiver/94874e0f1e35bd667bf3b7c3dc6416068779444f/biorxiv/exploratory_data_analysis/output/figures/preprint_headings.png){#fig:biorxiv_headings}

> BioRxiv has been notorious for having a bountiful supply of neuroscience preprints [@doi:10.7554/eLife.45133].

Reviewer: ```suggestion
Each preprint on bioRxiv has an author-selected topic area, and the predominant area in past reports has been neuroscience [@doi:10.7554/eLife.45133].
```

> Our findings support this trend when we quantified the number of preprints per article category within bioRxiv (Figure {@fig:biorxiv_categories}).

Reviewer: ```suggestion
Our analysis of the full text release of bioRxiv confirms this previous finding (Figure {@fig:biorxiv_categories}).
```

> Furthermore, majority of our category rankings match with previous studies [@doi:10.7554/eLife.45133; @doi:10.1001/jama.2017.21168].

Reviewer: ```suggestion
The author-selected topic area abundances that we found in the full text largely matched previous studies [@doi:10.7554/eLife.45133; @doi:10.1001/jama.2017.21168].
```

> The exception being microbiology as more papers were submitted than projected in 2018 [@doi:10.7554/eLife.45133] (Figure {@fig:biorxiv_categories}).

Reviewer: ```suggestion
One exception was microbiology, which has a larger share of preprints than in a previous report from 2018 [@doi:10.7554/eLife.45133] (Figure {@fig:biorxiv_categories}).
```

> Lastly, we quantified the type of result reported in each preprint.

Reviewer: ```suggestion
Authors also select from three article types when they upload their preprints.
```

> We found that the majority of preprints mention new results (Figure {@fig:biorxiv_headings}), which is consisted with previous findings [@doi:10.1001/jama.2017.21168].

Reviewer: ```suggestion
We found that most preprints are categorized as new results (Figure {@fig:biorxiv_headings}), which is consistent with previous findings [@doi:10.1001/jama.2017.21168].
```
